contents/docs/experiments/creating-an-experiment.mdx

```diff
@@ -44,43 +44,27 @@ Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In yo
-
-## Participant targeting
-
-By default, your experiment will target 100% of participants. If you want to target a more specific set of participants, or change the rollout percentage, you need to do this by changing the [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) for the feature flag used by your experiment.
-
-Below is a video showing how to navigate there:
-
-<ProductVideo
-    videoLight={SetExperimentLight} 
-    videoDark={SetExperimentDark}
-    alt="How to create an experiment in PostHog" 
-    classes="rounded"
-/>
+By default PostHog runs experiments at a user-level. This means that participants are assigned to a variant based on their user properties. However, if you've created [groups](/docs/product-analytics/group-analytics), you can also run [group-targeted experiments](/blog/running-group-targeted-ab-tests). This will test how a change affects your product at a group-level by providing the same variant to every member of a group.
 
 ## Experiment goal
 
-Setting your goal metric enables PostHog to calculate the impact of your experiment and if your results are [statistically significant](/docs/experiments/significance). You can select between a "trend" or "conversion funnel" goal.
+Setting your goal metric enables PostHog to calculate the impact of your experiment and if your results are [statistically significant](/docs/experiments/significance). You can select between a `trend` or `conversion funnel` goal.
 
-You set the minimum acceptable improvement below. This combines historical data from your goal metric with your desired improvement rate to show a prediction for how long you need to run your experiment to see statistical significant results.
+## Distribution and release conditions
 
-<ProductVideo
-    videoLight={ImprovementCalculatorLight} 
-    videoDark={ImprovementCalculatorDark}
-    alt="Minimum acceptable improvement calculator" 
-    classes="rounded"
-/>
+By default, your experiment will target 100% of participants. If you want to target a more specific set of participants, or change the rollout percentage, you need to do this by changing the [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) for the feature flag used by your experiment.
+
+You can do this by clicking on the **Manage distribution** or **Manage release conditions** button _after_ saving your experiment as a draft.
 
 > **Note:** If you select a server-side event, you may see a warning that no feature flag information can be detected with the event. To resolve this issue, see [step 2 of adding your experiment code and how to submit feature flag information with your events](/docs/experiments/adding-experiment-code#step-2-server-side-only-add-the-feature-flag-property-to-your-events).
 > 
```

Comment:
@jurajmajerik is this still true? I tried to replicate it but was unable to get it to show